tests/fixtures/project_with_tasks_that_handle_interrupts/loop_with_signal_handler.py

```diff
@@ -0,0 +1,18 @@
+import signal
```

Comment:
could you please move this specific file (and a copy of `pyproject.toml`) into a dir called `tests/fixtures/project_with_tasks_that_handle_sigterm`? this will better explain to other devs what the fixture is used for :)